src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubProtocolResolver.cs

```diff
@@ -11,23 +14,33 @@ namespace Microsoft.AspNetCore.SignalR.Internal
     public class DefaultHubProtocolResolver : IHubProtocolResolver
     {
         private readonly IOptions<HubOptions> _options;
+        private readonly ILogger<DefaultHubProtocolResolver> _logger;
+        private readonly IDictionary<string, IHubProtocol> _availableProtocols;
 
-        public DefaultHubProtocolResolver(IOptions<HubOptions> options)
+        public DefaultHubProtocolResolver(IOptions<HubOptions> options, IEnumerable<IHubProtocol> availableProtocols, ILogger<DefaultHubProtocolResolver> logger)
         {
             _options = options;
+            _logger = logger;
+            _availableProtocols = availableProtocols.ToDictionary(p => p.Name);
```

Comment:
case insensitive?

Reply:
I wasn't sure so I decided to leave it and wait and see if you'd comment ;P